src/nvim/mouse.c

```diff
@@ -1258,8 +1253,6 @@ int jump_to_mouse(int flags, bool *inclusive, int which_button)
   if (flags & MOUSE_SETPOS) {
     goto retnomove;                             // ugly goto...
   }
-  win_T *old_curwin = curwin;
-  pos_T old_cursor = curwin->w_cursor;
```

Comment:
What is the point of this change?

Reply:
No idea, probably failed to move it back to it's original position after a different change. Thanks for the thorough review. 

The jump happens at line 1359, so it seems this could be moved down to before that if statement.